packages/cozy-stack-client/src/NotesCollection.spec.js

```diff
@@ -0,0 +1,50 @@
+import NotesCollection from './NotesCollection'
+import CozyStackClient from './CozyStackClient'
+
+describe('NotesCollection', () => {
+  const stackClient = new CozyStackClient()
+  const collection = new NotesCollection(stackClient)
```

Comment:
The setup pattern seems better (where we do not instantiate suite global variables) since we are sure that tests do not interfere with each other.